apps/aecore/lib/aecore/txs/pool/worker.ex

```diff
@@ -59,6 +59,10 @@ defmodule Aecore.Txs.Pool.Worker do
         if tx_pool == updated_pool do
           Logger.info("Transaction is already in pool")
         else
+          # Broadcasting notifications for new transaction in a pool(per account and every)
+          Aehttpserver.Web.Notify.broadcast({:new_transaction_in_the_pool_per_account, Base.encode16(tx.data.from_acc)})
```

Comment:
can you combine these to `Aehttpserver.Web.Notify.broadcast_new_transaction_in_pool`

Reply:
do you mean to change the name of the function or something else?

Reply:
yes use specific functions for each broadcast and only have one function call here